Sources/HTTP/Message/HTTPBody.swift

```diff
@@ -160,10 +160,14 @@ public struct HTTPBody: Codable {
             stream.drain { buffer, upstream in
                 if let size = size {
                     guard data.count + buffer.count <= size else {
-                        throw HTTPError(identifier: "body-size", reason: "The body was larger than the max request.")
+                        throw HTTPError(identifier: "body-size", reason: "The body was larger than the request.")
```

Comment:
Error identifiers should be camel case

Reply:
also we should try to be cosistent with adding periods or not. I’ve been not including them but maybe we should since not every error message will be a single sentence

Reply:
I agree with the periods. Error identifiers have never been done one or another way throughout Vapor 3.

Reply:
I agree with the periods. Error identifiers have never been done one or another way throughout Vapor 3.